activerecord/lib/active_record/relation/batches.rb

```diff
@@ -28,10 +28,11 @@ module Batches
     # ==== Options
     # * <tt>:batch_size</tt> - Specifies the size of the batch. Default to 1000.
     # * <tt>:start</tt> - Specifies the primary key value to start from.
+    # * <tt>:end_at</tt> - Specifies the ending point for the batch processing.
```

Comment:
Less rhetorically: I just read all the API doc changes, and @kaspth's proposed alternate spelling, and was still unclear on whether it was inclusive or exclusive. I suggest we make that very explicit.

For that matter... have we considered and dismissed the idea of a `:range` option instead?